### PR TITLE
Changing GMUGeoJSONParser initWithURL initializer to accept externally hosted URLs.

### PR DESCRIPTION
**Pull request by kostrzynski**

### CODE REVIEW

File: src/Geometry/GMUGeoJSONParser.m [hunk 1]
```diff
@@ -106,7 +106,7 @@ - (instancetype)initWithStream:(NSInputStream *)stream {
 
 - (instancetype)initWithURL:(NSURL *)url {
   if (self = [super init]) {
-    _stream = [[NSInputStream alloc] initWithURL:url];
+    _data = [[NSData alloc] initWithContentsOfURL:url];
```
Comment (mountainvat) [inline src/Geometry/GMUGeoJSONParser.m]: Maybe OK for now but I'd like to move this expensive operation into the parse method later.

**Comment by domesticmouse** [comment]
Can you please bump the version number in the podspec?

**Comment by mountainvat** [comment]
I don't think we want to do that since it's not officially published.

**Comment by domesticmouse** [comment]
What do you mean? We have 2.0.0 in the podspec file on github, thus it is already cached. We need to bump it to 2.0.1 so those caches get invalidated.

File: src/Geometry/GMUGeoJSONParser.m (hunk 1 above)
Comment (domesticmouse) [inline src/Geometry/GMUGeoJSONParser.m]: Yeah, after using this, we are going to have to do something smarter before launching 2.0.

**Pull request closed without merging**